app/src/lib/server/indexingQueue.ts

```diff
@@ -0,0 +1,79 @@
+// Using dynamic import to avoid type errors before dependency is installed/built.
+// Lazy loader wrapper – if dependency missing we no-op.
+let SQSClient: any;
+let SendMessageCommand: any;
+async function loadSQS() {
+  if (SQSClient && SendMessageCommand) return { SQSClient, SendMessageCommand };
+  try {
+    const mod = await import('@aws-sdk/client-sqs');
+    SQSClient = mod.SQSClient;
+    SendMessageCommand = mod.SendMessageCommand;
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.warn('[indexingQueue] SQS client module not available; events will be skipped');
+    SQSClient = class {};
+    SendMessageCommand = class {};
+  }
+  return { SQSClient, SendMessageCommand };
+}
+
+// Indexing event minimal schema. Extend if you later need hierarchy info.
+export interface IndexingEvent {
+  action: 'upsert' | 'delete';
+  guid: string;
+  type?: string; // payload type
+  timestamp: string; // ISO string
+}
+
+let sqs: any | undefined;
+async function getClient() {
+  if (sqs) return sqs;
+  const queueRegion = process.env.INDEXING_QUEUE_REGION || 'fr-par';
+  const endpoint = process.env.INDEXING_QUEUE_ENDPOINT; // Scaleway SQS-compatible endpoint if provided
+  const accessKeyId = process.env.INDEXING_QUEUE_ACCESS_KEY || process.env.SCALEWAY_ACCESS_KEY || '';
```

Comment:
Environment als zod-Schema definieren wie in den Import-Skripten. SCALEWAY_* entfernen.